config/prow/update-jobs.yaml

```diff
@@ -28,4 +28,24 @@ roleRef:
 subjects:
 - apiGroup: rbac.authorization.k8s.io
   kind: Group
-  name: system:nodes
\ No newline at end of file
+  name: system:nodes
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: update-jobs-sa
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: update-jobs
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: ServiceAccount
+  name: update-jobs
+  namespace: prow
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ServiceAccount
+metadata:
+  name: update-jobs
+  namespace: prow
```

Comment:
the namespace we run pods in is `test-pods` and the namespace prow microservices run in `default` so this should be changed.